Jamfile

```diff
@@ -103,7 +133,7 @@ rule linking ( properties * )
 	}
 
 	# gcrypt libraries, if enabled
-	if <crypto>gcrypt in $(properties)
+	if <crypto>libgcrypt in $(properties)
```

Comment:
do you think libgcrypt is better than gcrypt? I realize I've been using the lib prefix. maybe it would be more conservative to preserve the gcrypt name and use crypto for libcrypto.